cmd/podman/containers/start.go

```diff
@@ -108,7 +111,35 @@ func start(cmd *cobra.Command, args []string) error {
 		startOptions.Stdout = os.Stdout
 	}
 
-	responses, err := registry.ContainerEngine().ContainerStart(registry.GetContext(), args, startOptions)
+	for _, f := range filters {
```

Comment:
This should only be done if a filters were specified.

Reply:
Where is that start protocol?

Reply:
Yes?